openedx/core/lib/block_structure/transformers.py

```diff
@@ -0,0 +1,100 @@
+"""
+Module for a collection of BlockStructureTransformers.
+"""
+from logging import getLogger
+
+from .exceptions import TransformerException
+from .transformer_registry import TransformerRegistry
+
+
+logger = getLogger(__name__)  # pylint: disable=C0103
+
+
+class BlockStructureTransformers(object):
+    """
+    Class for a collection of block structure transformers.
+    """
+    def __init__(self, transformers=None, usage_info=None):
+        """
+        Arguments:
+            transformers ([BlockStructureTransformer]) - List of transformers
+                to add to the collection.
+
+            usage_info (any negotiated type) - A usage-specific object
```

Comment:
If this is something shared across Transformers, don't they have to agree on what it is (or at least the shape of the container)?